index.ts

```diff
@@ -19,18 +22,21 @@ export default {
 			w.smartlook.api.push(arguments)
 		}
 
-		const { region = 'eu', version = 'nextgen', cookies = true } = params ?? {}
+		const { region = 'eu', version = 'nextgen', cookies = true, host } = params ?? {}
 
 		w.smartlook.api = []
-		w.smartlook('init', key, { region, cookies })
+		w.smartlook('init', key, { region, cookies, host })
 
 		const head = window.document.getElementsByTagName('head')[0]
 		const script = window.document.createElement('script')
+		const src =
+			version === 'nextgen'
+				? host ?? 'https://web-sdk.smartlook.com/recorder.js'
+				: 'https://rec.smartlook.com/recorder.js'
```

Comment:
Honestly, I am not a big fan of these double ternary operators because there are not readable at all. 

```suggestion
		let src =
			version === 'nextgen'
				? 'https://web-sdk.smartlook.com/recorder.js'
				: 'https://rec.smartlook.com/recorder.js'
			
			if(host !== undefined) {
			      src = host
			}
```

Also, it should be added to readme.  And note that script.src should contain `record.js`:

```
 c.src = 'https://<your-relay-proxy-host>/recorder.js';
```

Reply:
actually you might need to add also `https://` there based on relay proxy docs